Account/FirefoxAccountConfiguration.swift

```diff
@@ -106,7 +106,17 @@ public struct StageFirefoxAccountConfiguration: FirefoxAccountConfiguration {
 
     public let sync15Configuration: Sync15Configuration = StageSync15Configuration()
 
-    public let pushConfiguration: PushConfiguration = FirefoxBetaPushConfiguration()
+    public var pushConfiguration: PushConfiguration {
+        get {
+            #if MOZ_CHANNEL_RELEASE
+                return FirefoxStagingPushConfiguration()
```

Comment:
This is actually correct. When you flip the 'Use stage servers' button, all build flavours should talk to stage.